distribution/docs/src/main/jdocs/content/_security-framework/security-pdp-authz-realm-contents.adoc

```diff
@@ -0,0 +1,42 @@
+:title: Security PDP AuthZ Realm
+:type: subSecurityFramework
+:status: published
+:parent: Security PDP
+:order: 00
+:summary: Security PDP AuthZ Realm.
+
+The ${ddf-security} PDP AuthZ Realm exposes a realm service that makes decisions on authorization requests using the attributes stored within the metacard to determine if access should be granted.
+This realm can use XACML and will delegate decisions to an external processing engine if internal processing fails.
+Decisions are first made based on the "match-all" and "match-one" logic.
+Any attributes listed in the "match-all" or "match-one" sections will not be passed to the XACML processing engine and they will be matched internally.
+It is recommended to list as many attributes as possible in these sections to avoid going out to the XACML processing engine for performance reasons.
+If it is desired that all decisions be passed to the XACML processing engine, remove all of the "match-all" and "match-one" configurations.
+The configuration below provides the mapping between user attributes and the attributes being asserted - one map exists for each type of mapping (each map may contain multiple values).
+
+Match-All Mapping:: This mapping is used to guarantee that all values present in the specified metacard attribute exist in the corresponding user attribute.
+Match-One Mapping:: This mapping is used to guarantee that at least one of the values present in the specified metacard attribute exists in the corresponding user attribute.
+
+===== Configuring the Security PDP AuthZ Realm
+
+. Navigate to the ${admin-console}.
+. Select *${ddf-security}* Application.
+. Select *Configuration* tab.
+. Select *Security AuthZ Realm*.
+
+.Security PDP AuthZ Realm Settings
+[cols="1,1,6" options="header"]
+|===
+
+|Configuration Name
+|Default Value
+|Additional Description
+
+|Match-All Mappings
+| 
+|These map user attributes to metacard security attributes to be used in "Match All" checking. All the values in the metacard attribute must be present in the user attributes in order to "pass" and allow access. These attribute names are case-sensitive.
+
+|Match-One Mappings
+|
+|These map user attributes to metacard security attributes to be used in "Match One" checking. At least one of the values from the metacard attribute must be present in the corresponding user attribute to "pass" and allow access. These attribute names are case-sensitive.
+
```

Comment:
There are some other config fields we're missing here for the PDP

Reply:
This is documented somewhere else, so I converted this to a link to that section.